### ISSUE TITLE
mac SDL1 doesn't respect fullscreen=original

### ISSUE THREAD

**Issue by emendelson** [joncampbell123/dosbox-x#983]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - Operating system
 - DOSBox-X release version or commit SHA1 this issue occurred on
 - Used configuration, e.g. ```dosbox.conf```

**Additional context**

**Comment by joncampbell123** [comment]
I'm not experiencing the host key issue on Mojave. I don't have a Macbook Pro with High Sierra to test that on.

I will be releasing the next version in the next 24 hours.

A discussion here on what is expected to happen in fullscreen mode is welcome.

**Comment by emendelson** [comment]
Here's some thoughts on full-screen:

1. If fullresolution=original, then the output should be centered in the middle of the full screen. This doesn't always happen with OpenGL, though it seems reliable with Surface. (I think in OpenGL, the full-screen, fullresolution=original, image goes to the upper left corner. It should be centered.)

2. Happy New Year!

**Comment by emendelson** [comment]
Yes - I know that about output=surface - it's one reason I use it for my WordPerfect project, where I use custom VGA fonts with 43, 50, etc., lines. They only look good when NOT scaled. (And while we're on the phone, I still hope that the long-term plan for scalable fonts instead of bitmaps when using standard codepages is still possible someday...)
